docs/contributing/building/packaging.md

```diff
@@ -16,14 +16,14 @@ to install build depencies in your OS.
 
```

Comment:
not sure why I cannot comment on the beginning of the file and I know you have not been editing it but it says 
"local building building and"
is it supposed to say local build?